src/Apps/__stories__/Apps.story.tsx

```diff
@@ -14,11 +15,22 @@ storiesOf("Apps", module)
     )
   })
   .add("Artist Page", () => {
+    const filterState = new FilterState()
+    // TODO: This currently only changes the UI, the first fetch isn’t taking
+    //       this state into account.
+    filterState.state.for_sale = true
     return (
       <StorybooksRouter
         routes={artistRoutes}
         initialRoute="/artist2/andy-warhol"
-        initialState={{ mediator: { trigger: x => x } }}
-      />
+        initialAppState={{ mediator: { trigger: x => x } }}
+        initialState={[filterState]}
+        subscribeTo={[FilterState]}
+      >
+        {x => {
+          // tslint:disable-next-line:no-console
+          console.log(x.state)
+        }}
```

Comment:
Just like with the `Subscribe` API, pass a render prop as `children` to get state updates.

One thing to note is that it currently also invokes the callback on the first render, something which we probably don’t need or even want?